src/pg/sql/45_observatory_mvt.sql

```diff
@@ -263,17 +263,66 @@ BEGIN
 END
 $$ LANGUAGE plpgsql PARALLEL RESTRICTED;
 
+CREATE OR REPLACE FUNCTION cdb_observatory.OBS_GetMCDates(
```

Comment:
Are you using this function?

Reply:
Or is a public function to be used to pass the months to the obs_getmcdo... function?

Reply:
Is a helper function that can be used to get the months needed by the `obs_getmcdomvt` function